t/api/08-jobtemplates.t

```diff
@@ -507,6 +508,114 @@ $t->post_ok(
 );
 is($job_templates->search({prio => -5})->count, 0, 'no rows affected');
 
+# test the YAML export
+# Test validation
+my %yaml;
+is_deeply(OpenQA::WebAPI::Controller::API::V1::JobTemplate::validate_yaml($t, \%yaml, 1), [], 'Empty YAML is fine')
```

Comment:
The fact that you have to type `OpenQA::WebAPI::Controller::API::V1::JobTemplate::validate_yaml` here suggests to me that this might be better off being a helper. 😉

Reply:
Then it's just `$t->app->validate_yaml(...)`.